<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registrationWindow</class>
 <widget class="QDialog" name="registrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>224</width>
    <height>281</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>224</width>
    <height>281</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>224</width>
    <height>281</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Регистрация</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>135</width>
     <height>160</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Введите логин</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="loginEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Введите пароль</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Повторите пароль</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="repeatPasswordEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="registrationButton">
      <property name="text">
       <string>Зарегистрироваться</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
